source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -3,45 +3,70 @@ pragma solidity ^0.4.13;
 import 'reporting/IReputationToken.sol';
 import 'reporting/IUniverse.sol';
 import 'reporting/IReportingWindow.sol';
+import 'libraries/math/SafeMathUint256.sol';
+import 'reporting/Reporting.sol';
 
 
 contract MarketFeeCalculator {
+    using SafeMathUint256 for uint256;
+
     mapping (address => uint256) private shareSettlementPerEthFee;
     mapping (address => uint256) private validityBondInAttoeth;
+    mapping (address => uint256) private targetReporterGasCosts;
+
+    uint256 private constant DEFAULT_VALIDITY_BOND = 1 ether;
```

Comment:
I think Joey was right that this is probably too high ($300).  Maybe `1 ether / 100` instead?